darwin/importer/importer.py

```diff
@@ -581,37 +716,106 @@ def _import_properties(
+            f"Performing {len(update_properties)} property update(s):", style="info"
+        )
         for full_property in update_properties:
+            if full_property.granularity.value == "item":
+                console.print(
+                    f"- Updating item-level property '{full_property.name}' with new value: {full_property.property_values[0].value}",
+                )
+            else:
+                console.print(
+                    f"- Updating property '{full_property.name}' of type {full_property.type}",
+                )
+            prop = client.update_property(
+                team_slug=full_property.slug, params=full_property
+            )
+            updated_properties.append(prop)
+
+    # get latest team properties
+    team_properties_annotation_lookup, team_item_properties_lookup = (
+        _get_team_properties_annotation_lookup(client, dataset.team)
+    )
+
+    create_properties = []
+    update_properties = []
+
+    # Create/Update properties from item_properties arg
+    create_properties, update_properties = _create_update_item_properties(
+        _normalize_item_properties(item_properties),
+        team_item_properties_lookup,
+        create_properties,
+        update_properties,
+        client,
+    )
+
+    if create_properties:
+        console.print(f"Creating {len(create_properties)} properties:", style="info")
+        for full_property in create_properties:
+            if full_property.granularity.value == "item":
+                console.print(
+                    f"- Creating item-level property '{full_property.name}' of type: {full_property.type}"
+                )
             console.print(
-                f"Updating property {full_property.name} ({full_property.type})",
-                style="info",
+                f"- Creating property '{full_property.name}' of type {full_property.type}",
             )
+            prop = client.create_property(
+                team_slug=full_property.slug, params=full_property
+            )
+            created_properties.append(prop)
+
+    if update_properties:
+        console.print(
+            f"Performing {len(update_properties)} property update(s):", style="info"
+        )
+        for full_property in update_properties:
+            if full_property.granularity.value == "item":
+                console.print(
+                    f"- Updating item-level property '{full_property.name}' with new value: {full_property.property_values[0].value}"
+                )
+            else:
+                console.print(
+                    f"- Updating property {full_property.name} ({full_property.type})",
+                )
             prop = client.update_property(
                 team_slug=full_property.slug, params=full_property
             )
             updated_properties.append(prop)
 
     # get latest team properties
     team_properties_annotation_lookup = _get_team_properties_annotation_lookup(
```

Comment:
this returns the item level prop lookup as well, so this check
```
        if (prop_name, annotation_class_id) not in team_properties_annotation_lookup:
            continue
```
will always be entered it seems

Reply:
Good catch, working on a fix now